hack/cluster-deploy.sh

```diff
@@ -56,6 +56,17 @@ if [[ "$KUBEVIRT_PROVIDER" =~ os-* ]] || [[ "$KUBEVIRT_PROVIDER" =~ okd-* ]]; th
     _kubectl adm policy add-scc-to-user privileged admin
 fi
 
+if [[ $TARGET =~ .*sriov.* ]]; then
```

Comment:
please use KUBEVIRT_PROVIDER. TARGET should be used in automation/test.sh only. In most cases KUBEVIRT_PROVIDER is set to the same value over there.